src/access/profiling/manager.py

```diff
@@ -49,15 +50,46 @@ def parse(self) -> xr.Dataset:
         )
 
 
+class ProfilingExperimentStatus(Enum):
+    """Enumeration representing the status of a profiling experiment."""
+
+    NEW = 1
+    RUNNING = 2
+    DONE = 3
+
+
+class ProfilingExperiment:
+    """Represents a profiling experiment.
+
+    Args:
+        path (Path): Path to the experiment directory.
+    """
+
+    path: Path  # Path to the experiment directory
+    status: ProfilingExperimentStatus = ProfilingExperimentStatus.NEW  # Status of the experiment
+
+    def __init__(self, path: Path) -> None:
+        self.path = path
+
+
 class ProfilingManager(ABC):
     """Abstract base class to handle profiling data and workflows.
 
     This high-level class defines methods to parse different types of profiling data. Currently,
     it supports parsing and plotting scaling data.
+
+    Args:
```

Comment:
Hmm - should this be `members` (instead of `Args`)?

Reply:
No, `Args` is correct.  One can document the `__init__`  arguments in the class docstring instead of the function itself. I personally prefer it over documenting the function itself, but open for discussion.